test/3D_laplacian.jl

```diff
@@ -1,21 +1,36 @@
-using DiffEqOperators
-
+using DiffEqOperators, DifferentialEquations, ProgressMeter
+function main()
```

Comment:
why is this in a function?

Reply:
to avoid the problem of global variables being mutated in a loop not working